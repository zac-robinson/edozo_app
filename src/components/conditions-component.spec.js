import React from 'react';
import { shallow } from 'enzyme';

import Conditions from './conditions-component';

describe('Conditions component', () => {
  it('should render correctly', () => {
    const conditionsProp = {
      icon: 'testIcon',
      main: 'name of icon',
      description: 'its probably raining'
    };
    const component = shallow(<Conditions conditions={conditionsProp} />);

    expect(component).toMatchSnapshot();
  });
});
