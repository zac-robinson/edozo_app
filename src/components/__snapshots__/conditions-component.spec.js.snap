// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Conditions component should render correctly 1`] = `
<styled.div>
  <img
    alt="name of icon"
    height="50"
    src="http://openweathermap.org/img/w/testIcon.png"
    width="50"
  />
  <styled.span>
    its probably raining
  </styled.span>
</styled.div>
`;
